src/components/Footer/index.tsx

```diff
@@ -166,22 +166,12 @@ export const Footer: FunctionComponent<FooterPropsType> = props => {
           </FooterSectionLinks>
 
           <FooterSection>
-            {props.copyText ? (
-              <Text textSize="s">{props.copyText}</Text>
-            ) : props.theme?.name === 'su2c' ? (
-              <Text textSize="s">
-                Stand Up To Cancer and Stand Up To Cancer Brand Marks are registered trademarks of the Entertainment
-                Industry Foundation. Cancer Research UK is a registered charity in England and Wales (1089464), Scotland
-                (SC041666), the Isle of Man (1103) and Jersey (247). A company limited by guarantee. Registered company
-                in England and Wales (4325234) and the Isle of Man (5713F). Registered address: 2 Redman Place, London,
-                E20 1JQ. Donations will be made to Cancer Research UK in support of the Stand Up To Cancer campaign.
-              </Text>
+            {typeof props.middleSection === 'string' ? (
```

Comment:
logic should be:
```
!!middleSection ? 
typeof middleSection === 'string' ? 
<Text>{middleSection}<Text> 
: {middleSection} 
: theme.siteConfig.footerCopyText
```

you don't need to check for a value on theme, all themes have to follow the same shape so by type definitions it has to be there